scripts/install-dependencies.sh

```diff
@@ -14,19 +14,15 @@ show_help() {
   echo
   echo "Available options:"
   echo
-  echo "-l --lhelper              Install tools required by LHelper and doesn't"
-  echo "                          install external libraries."
   echo "   --debug                Debug this script."
   echo
 }
 
 main() {
-  local lhelper=false
-
   for i in "$@"; do
     case $i in
       -s|--lhelper)
-        lhelper=true
+        echo "--lhelper has been deprecated and will be removed in a future release"
         shift
         ;;
```

Comment:
Should we just remove this?

Reply:
I kept it to inform people of the change since not many people follow the development of projects

Reply:
I don't think the switch was ever documented in our README.
Considering that the switch only prints out a message, it might be better to just abort the script execution.